src/GAP/wrappers.jl

```diff
@@ -346,6 +351,8 @@ GAP.@wrap Random(x::GapObj, y::GapObj)::GAP.Obj
 GAP.@wrap Range(x::GapObj)::GapObj
 GAP.@wrap RecognizeGroup(x::GapObj)::GapObj
 GAP.@wrap ReduceCoeffs(x::GapObj, y::GapObj)
+GAP.@wrap RelativeOrder(x::GapObj)::Int
+GAP.@wrap RelativeOrderOfPcElement(x::GapObj, y::GapObj)::Int
```

Comment:
Hmmm... I think these can (at least in principle) return large integers, i.e. which don't fit into an `Int`?
```suggestion
GAP.@wrap RelativeOrder(x::GapObj)::GapInt
GAP.@wrap RelativeOrderOfPcElement(x::GapObj, y::GapObj)::GapInt
```

Same for `LeadingExponent`. Though I might be wrong, gotta test this at some point...